library/core/class.controller.php

```diff
@@ -1809,7 +1809,8 @@ public function renderMaster() {
                     // style.css and admin.css deserve some custom processing.
                     if (in_array($CssFile, $CssAnchors)) {
                         // Grab all of the css files from the asset model.
-                        $CssFiles = $AssetModel->getCssFiles($ThemeType, ucfirst(substr($CssFile, 0, -4)), $ETag);
+                        $dummy = null;
+                        $CssFiles = $AssetModel->getCssFiles($ThemeType, ucfirst(substr($CssFile, 0, -4)), $ETag, $dummy, $this->Theme);
```

Comment:
Just use `null` inline rather than setting a dummy variable first.

Reply:
I did that in the first place but then it dumped with:  'Cannot pass parameter null by reference', because `$AssetModel->getCssFiles` expects a reference for parameter 4 (I was not brave enough to change that. I can't see the implications), but I can change that if you say so. Can you advise?

Reply:
You're correct. I went digging and as much as this method signature is extraordinarily poorly designed, I dare not alter it now given the costs & possible repercussions of that. Now I'm wondering if we're approaching this the wrong way from the start. I'll have a developer take a look at this or I'll circle back later when I can unwind this a bit more.

Reply:
As an aside, I traced that pass-by-ref to a change by Tim in 2015, which I assume means our cloud infrastructure is doing something with it.

Reply:
The convention is `$_` as a dummy argument and you don't have to declare it first.